src/app.rs

```diff
@@ -0,0 +1,56 @@
+
+use std::thread;
+use std::time::Duration;
+use std::fs::File;
+use std::process::Command;
+use std::io::Read;
+
+use conf::Config;
+use time::get_time;
+use item::ItemKind;
+
+pub fn start(mut conf: Config) {
+    // We would deamonize here if necessary
+
+    loop {
+        loop {
+            if let Some(c) = conf.items.peek() {
+                if c.next_time > get_time().sec {
+                    break;
+                }
+            } else {
+                break;
+            }
+
+
+            let mut item = conf.items.pop().unwrap();
+            let clone = item.clone();
+            item.next_time = get_time().sec + item.interval;
```

Comment:
If interval is 0, this will still not execute it again in the same loop, however it will execute it again in the next, with practically no time between. That is a bug to me, so I will make an issue.